code/modules/mob/living/silicon/ai/ai_defense.dm

```diff
@@ -1,5 +1,5 @@
 /mob/living/silicon/ai/attack_alien(mob/living/carbon/alien/humanoid/M)
-	if(!SSticker)
+	if(SSticker.current_state < GAME_STATE_PLAYING)
 		to_chat(M, "You cannot attack people before the game has started.")
```

Comment:
[Not a review comment]

God I wonder what the old devs were thinking.

Reply:
You think they were thinking? /j

Reply:
Why is this even a check, like how often are AI's being attacked by xenos before a roundstart?